File: demo/src/main/java/com/example/demo/model/Monitorador.java
package com.example.demo.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
public class Monitorador{

  @Id
  @GeneratedValue(strategy = GenerationType.AUTO)
  private Long id;

  @Column(nullable = false)
  private String nome;

  @Column(nullable = false)
  private String tipoPessoa;

  private String cpf;

  private String rg;

  private String cnpj;

  private String inscricaoEstadual;

  @Column(nullable = false)
  private String email;

  @Column(nullable = false)
  private String contato;

}

File: src/app/classes/monitorador.ts
export class Monitorador {
  id: number;
  nome: string;
  cpf: string;
  rg: string;
  email: string;
  cnpj: string;
  inscricaoEstadual: string;
  contato: string;
  dataNascimento: string;
  ativo: string


  constructor(
    id:number,
    nome: string,
    cpf: string,
    rg: string,
    email: string,
    cnpj: string,
    inscricaoEstadual: string,
    contato: string,
    dataNascimento: string,
    ativo: string

    ){
      this.id = id;
      this.nome = nome;
      this.cpf = cpf;
      this.rg = rg;
      this.email = email;
      this.cnpj = cnpj;
      this.inscricaoEstadual = inscricaoEstadual;
      this.contato = contato;
      this.dataNascimento = dataNascimento;
      this.ativo = ativo
  }
}
